renovate-config.json5

```diff
@@ -114,6 +114,26 @@
       groupName: 'page kit monorepo',
       groupSlug: 'page-kit',
     },
+    {
+      // group Tool Kit packages together
+      matchPackagePrefixes: [
+        '@dotcom-tool-kit/',
```

Comment:
**issue**: I think you'll also need to include the `dotcom-tool-kit` package here as well?

Reply:
YES Good catch!

Reply:
Done in [9cc43cb](https://github.com/Financial-Times/renovate-config-next/pull/119/commits/9cc43cb2023005020ca4a54fa982968fcb27079e).